core/market/decentralized/src/market.rs

```diff
@@ -41,8 +42,8 @@ pub enum MarketInitError {
 /// Structure connecting all market objects.
 pub struct MarketService {
     pub matcher: Arc<Matcher>,
-    pub provider_negotiation_engine: Arc<ProviderNegotiationEngine>,
-    pub requestor_negotiation_engine: Arc<RequestorNegotiationEngine>,
+    pub provider_engine: Arc<ProviderNegotiationEngine>,
```

Comment:
👍

Reply:
```suggestion
    pub provider_engine: Arc<ProviderNegotiatior>,
```
maybe, consistent with `Matcher` ?

Reply:
Negotiatior is someone who negotiates. It sounds like agent job...

Reply:
True.
So maybe `ProviderBroker` :)
https://en.wikipedia.org/wiki/Negotiation

Reply:
Ok :D